core/src/gl/gl_fwd.h

```diff
@@ -0,0 +1,334 @@
+#pragma once
+
+#ifdef PLATFORM_ANDROID
```

Comment:
Why is this needed here? If this is not needed we can we move this file to `include/`.